app/src/main/java/com/door43/translationstudio/core/TargetTranslationMigrator.java

```diff
@@ -347,7 +346,66 @@ private static boolean v2( File path) throws Exception {
         return true;
     }
 
-   /**
+    /**
+     * convert projects from USX to USFM
+     * @param targetTranslations
+     * @return
+     */
+    public static boolean migrateFromUSXtoUSFM(TargetTranslation[] targetTranslations) {
```

Comment:
@neutrinog - Making migrateChunkChanges private breaks a lot of things.  WIll look into it.